.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+# Description
+
+Please include a summary of the changes and the related issue (if applicable). Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+Additional dependencies intoduced.
+- Dependency 1
+- Dependency 2
+
+## Type of change
+
+Please select what type of change this is.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New benchmark
+    - [ ] Non-breaking change which entirely uses exisiting classes, structure etc
+    - [ ] Breaking change which has implemented new/modified classes etc
+- [ ] New feature 
+    - [ ] Non-breaking change which adds functionality
+    - [ ] Breaking change fix or feature that would cause existing functionality to not work as expected
+
+## Other changes
+
+- [ ] This change requires a documentation update
+- [ ] (If Benchmark) This requires additional data that can be obtained from:
+    - Benchmark data 1
+    - Benchamrk data 2
+
+# Testing
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- Test A
+- Test B
+- etc
+
+**Test Configuration**:
```

Comment:
I would say this additional detail is on the test configuration is not needed.

Reply:
Removed